### PR TITLE
Upload QAT Gazette Feb 2018

### PR DESCRIPTION
**Pull request by JBHyperion**
### Description

Adds Feb 2018 QAT Gazette blog

### Status

Waiting on a confirmation of MWR ending date
All else finished, Pending Review

### CODE REVIEW

File: news/2018-02-18-qat-gazette-february-2018.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+[- Frontier -](https://osu.ppy.sh/users/4314710), [ailv](https://osu.ppy.sh/users/6933054), [AyanokoRin](https://osu.ppy.sh/users/5376866), [Bubblun](https://osu.ppy.sh/users/6191653), [dsco](https://osu.ppy.sh/users/4330511), [ezek](https://osu.ppy.sh/users/180241), [Hailie](https://osu.ppy.sh/users/5226970), [iYiyo](https://osu.ppy.sh/users/3919785), [Kuron-Kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [mancuso_JM_](https://osu.ppy.sh/users/521568), [Namki](https://osu.ppy.sh/users/5248582), [Nevo](https://osu.ppy.sh/users/7451883), [Noffy](https://osu.ppy.sh/users/1541323), [pinataman](https://osu.ppy.sh/users/1235485), [Realazy](https://osu.ppy.sh/users/918297), [Sotarks](https://osu.ppy.sh/users/4452992), [TheKingHenry](https://osu.ppy.sh/users/5128277), [timemon](https://osu.ppy.sh/users/2072005), [UndeadCapulet](https://osu.ppy.sh/users/2523533) and [Wishkey](https://osu.ppy.sh/users/1101704)!
```
Comment (Xinnoh) [inline news/2018-02-18-qat-gazette-february-2018.md]: electoz + gabe is missing

File: news/2018-02-18-qat-gazette-february-2018.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Some are returning veterans, some are stepping up for the first time, but all are working towards the same goal of bringing high quality beatmaps to the Ranked section. Many congratulations to you all on the start of another exciting journey in mapping and modding!
+
+### Beatmap Nominator retirements
+
+Sadly time waits for no circle, drum, fruit or piano inspector, and whilst many are just beginning their journey, others are taking a well-deserved break. Please offer your appreciation to [Bonsai](https://osu.ppy.sh/users/987334), [Monstrata](https://osu.ppy.sh/users/2706438), [MrSergio](https://osu.ppy.sh/users/2581696), [Rizia](https://osu.ppy.sh/users/1367570), [sheela](https://osu.ppy.sh/users/1138027) and [Zero__wind](https://osu.ppy.sh/users/1822830) for the osu! game mode, as well as [Litharrale](https://osu.ppy.sh/users/3189995) for the osu!mania game mode. Their hard work and lasting contributions have ensured players can enjoy a more varied and higher quality osu! experience for years to come.
```
Comment (Xinnoh) [inline news/2018-02-18-qat-gazette-february-2018.md]: OzzyOzrock also left

File: news/2018-02-18-qat-gazette-february-2018.md (hunk 1 above)
Comment (JBHyperion) [reply news/2018-02-18-qat-gazette-february-2018.md]: I thought 23 sounded familiar but forgot exisitng othermode people. Added them in a new sentence.

File: news/2018-02-18-qat-gazette-february-2018.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Additionally, [Electoz](https://osu.ppy.sh/users/6485263) and [Gabe](https://osu.ppy.sh/users/654108) have joined the ranks of the osu! Beatmap Nominators, after previously demonstrating their proficiency in osu!catch and osu!taiko, respectively. Well done to them both!
+
+### Beatmap Nominator retirements
+
+Sadly time waits for no circle, drum, fruit or piano inspector, and whilst many are just beginning their journey, others are taking a well-deserved break. Please offer your appreciation to [Bonsai](https://osu.ppy.sh/users/987334), [Monstrata](https://osu.ppy.sh/users/2706438), [MrSergio](https://osu.ppy.sh/users/2581696), [OzzyOzrock](https://osu.ppy.sh/users/465153), [Rizia](https://osu.ppy.sh/users/1367570), [sheela](https://osu.ppy.sh/users/1138027) and [Zero__wind](https://osu.ppy.sh/users/1822830) for the osu! game mode, as well as [Litharrale](https://osu.ppy.sh/users/3189995) for the osu!mania game mode. Their hard work and lasting contributions have ensured players can enjoy a more varied and higher quality osu! experience for years to come.
```
Comment (JBHyperion) [inline news/2018-02-18-qat-gazette-february-2018.md]: Also removed OzzyOzrock. RIP Ozzy

File: news/2018-02-25-qat-gazette-february-2018.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Acceptable content rule clarification
+
+A current hot topic in the osu! Ranking Criteria has been the unclear ruling on what we deem to be **"inappropriate or not-safe-for-work content"**. With the recent allowance/removal of some potentially borderline cases, it has become necessary to clarify and expand upon the current rule wording. This will aim to satisfy both mappers who wish to continue creating more mature-themed content, as well as players and moderators who wish to ensure osu! remains a safe and enjoyable environment for all.
+
+A first draft of the [rule change proposal](https://gist.github.com/Ephemeralis/b253dedf2fd87bbb6c154e483d449f2a) has been opened for a round of community discussion. Here, anyone can comment on the planned changes, as well as suggest further revisions. Included are guidelines on how to better decide if a song is appropriate for osu!, as well as clarification on what happens if a track is deemed to be unacceptable.
```
Comment (Ephemeralis) [inline news/2018-02-25-qat-gazette-february-2018.md]: probably want to change this to reference the fact that this got implemented and link to the thing in the ranking criteria or something

Comment (JBHyperion) [reply news/2018-02-25-qat-gazette-february-2018.md]: Switched this out with the RC article and the fact that it's now finalised

**Pull request merged**